doc/manual/source/data/patches.rst

```diff
@@ -107,4 +107,4 @@ The ouput of the ``diff`` command for that same modification will be as show nex
 	parktype: "Garden" -> "Park"
 	the_geom: -122.8456209953915,42.30337924298179 [-122.84288079012158,42.30337829301001] (-122.84288079012158,42.30406953954324) -122.84062871947361,42.303377484342896 -122.84402124697434,42.29758391876194 [-122.8425462600999,42.29842718304132] (-122.84348886900885,42.29846488739768) -122.84403459217438,42.299849758882665 (-122.84467141796014,42.29932750407095) -122.84553477839586,42.29898797978287 [-122.8455264791633,42.29813114136922] (-122.84516200371851,42.2974273267172) -122.84562615512898,42.29822488745995 -122.84562739414677,42.29869863907225 -122.84562865934777,42.29887914225196 -122.84562746190791,42.29938631374956 -122.84562038858874,42.29949283132804 -122.84562093251843,42.30009514260314 -122.84561954067343,42.300597171392674 -122.84562020542222,42.3019375538702 -122.84562986430872,42.302242481706564 -122.84562420177575,42.3031702527034 -122.84562286340557,42.30323277320125 -122.8456209953915,42.30337924298179 @-122.84070544530037,42.29863971301672 -122.84066916748334,42.298658265906056 -122.84114150211431,42.29917259286536 -122.84175134391927,42.2988708450611 -122.84154879818728,42.298647383830684 -122.84162235642542,42.29829337633362 -122.8412561201354,42.297896172348146 -122.8417060301776,42.29766808625502 -122.8412177383454,42.29741710209413 -122.84103109529498,42.29732116850443 -122.84088785596595,42.29783772129766 -122.8398848619697,42.29795997303185 -122.83999152040946,42.29828853706577 -122.84070544530037,42.29863971301672
 
-Since the syntax used for describing differences stored in a patch is similar to that of the ``diff`` command -except, as mentioned above, in the case of geometries, where ``diff`` shows a full list of all coordinates, while the patch summary just shows the number of affected points-, checking the documentation for the ``diff`` command is recommended.
\ No newline at end of file
+Since the syntax used for describing differences stored in a patch is similar to that of the ``diff`` command--except, as mentioned above, in the case of geometries, where ``diff`` shows a full list of all coordinates, while the patch summary just shows the number of affected points--checking the documentation for the ``diff`` command is recommended.
```

Comment:
This change doesn't do anything different than the original. The text between `-except` and `points-` should probably be contained within parentheses `( )`, as it is sort of a note inside the other text. It's a rather long note, but it's just an exception to the statement being made if the entire chunk between the single dashes were to be removed.